app/packages/rust_sdk/lib/effektio_flutter_sdk.dart

```diff
@@ -233,4 +257,37 @@ class EffektioSdk {
   ffi.CreateGroupSettings newGroupSettings(String name) {
     return _api.newGroupSettings(name);
   }
+
+  Future<bool> reportBug(String text, String label, bool withLog) async {
+    const url = String.fromEnvironment(
+      'RAGESHAKE_URL',
+      defaultValue: 'https://rageshake.acter.global/api/submit',
+    );
+    const username = String.fromEnvironment(
+      'RAGESHAKE_USERNAME',
+      defaultValue: 'acter',
+    );
+    const password = String.fromEnvironment(
+      'RAGESHAKE_PASSWORD',
+      defaultValue: 'global',
+    );
```

Comment:
Please use defaults for a localhost setup, not the ones we were to use in the actual setup (and I know have to change, as they've been leaked ;P )

Reply:
Fixed by https://github.com/acterglobal/a3/pull/488/commits/b14d323fbe0bcf467b22b3be661ca795f569104a